### PR TITLE
initial travis-ci support

### PR DESCRIPTION
**Pull request by yetist**
Need to test.

### CODE REVIEW

**Comment by yetist** [comment]
@raveit65
Can you enable travis for this repo?

**Comment by raveit65** [comment]
@yetist 
done, it's enabled.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+sudo: required
+language: bash
+services:
+  - docker
+
+# Trigger only on the travis branch
+# Remove the lines if every thing is ok.
+branches:
+  only:
+  - travis
```
Comment (raveit65) [inline .travis.yml]: This can be removed before merging?

**Comment by yetist** [comment]
@raveit65 

It is ready to merge now.

**Comment by sc0w** [comment]
@mate-desktop/core-team do you know why travis never begin to work with https://github.com/mate-desktop/mate-menus/commit/351af7ed0622b0b9dc0f6b79bb0a74f0a5be3559 ?

(branch `cppcheck_defines` in `mate-menus`)

**Pull request merged**